docker/jenkins/Makefile

```diff
@@ -2,13 +2,22 @@ BASE_FILE=docker-compose.yml
 OVERRIDE_FILE=docker-compose.override.yml
 SERVICE=ci
 SETUP_SCRIPT=../nvidia/setup_nvidia_docker_compose.py
+ENV_VARIABLES=CUDA_VERSION GPU_ARCH
```

Comment:
OK so looking at this now we can specify the cuda version and architecture we are using on Jenkins

Reply:
Yes.  This would enable having both a CUDA 8.0 and CUDA 9.0 build on fetnat01.